### PR TITLE
[hotfix] Correct the doc related to restart time tracking

### PR DESCRIPTION
**Pull request by 1996fanrui**
Backport #802  to 1.8, the doc can be updated timely

### CODE REVIEW

File: docs/content/docs/custom-resource/autoscaler.md [hunk 1]
```diff
@@ -162,12 +162,14 @@ Setting `job.autoscaler.target.utilization.boundary: "0.2"` means that we allow
 ### Target catch-up duration and restart time
 
 When taking scaling decisions the operator need to account for the extra capacity required to catch up the backlog created during scaling operations.
-The amount of extra capacity is determined automatically by the following 2 configs:
+The amount of extra capacity is determined automatically by the following 3 configs:
```
Comment (gyfora) [inline docs/content/docs/custom-resource/autoscaler.md:165]: maybe we can remove the number simply :
` determined automatically by the following configs`

**Review by gyfora — APPROVED** [review]
aside from a minor comment, looks good

File: docs/content/docs/custom-resource/autoscaler.md (hunk 1 above)
Comment (gyfora) [reply docs/content/docs/custom-resource/autoscaler.md:165]: Oh this is only a backport, then nevermind :)

Comment (1996fanrui) [reply docs/content/docs/custom-resource/autoscaler.md]: > Oh this is only a backport, then nevermind :)

Sorry, I didn't notice this comment.

If so, the 1.8 doc could remove this number, and we don't need to update the doc for main branch.

**Review by mxm — APPROVED** [review]

**Comment by 1996fanrui** [comment]
Thanks for the review, merging~

**Pull request merged**